homeassistant/components/daikin/climate.py

```diff
@@ -247,7 +247,9 @@ async def async_update(self):
 
     async def async_turn_on(self):
         """Turn device on."""
-        await self._api.device.set({})
+        await self._api.device.set(
+            {HA_ATTR_TO_DAIKIN[ATTR_HVAC_MODE]: HA_STATE_TO_DAIKIN[HVAC_MODE_HEAT_COOL]}
```

Comment:
Turn on is not the same as set to `HVAC_MODE_HEAT_COOL`.

The current approach on 69-based units works as long as the mode isn't `off`, https://bitbucket.org/mustang51/pydaikin/src/24b10556189ee119db34a2f070a223ed82e7f297/pydaikin/daikin_brp069.py#lines-123:129

Reply:
Is this fix for SKYFi units?

I would guess that this is the issue: https://bitbucket.org/mustang51/pydaikin/src/24b10556189ee119db34a2f070a223ed82e7f297/pydaikin/daikin_skyfi.py#lines-142:143

Reply:
The Daikin AC Type is:  daikin ftx35j2v1b

The issue is described here: https://github.com/home-assistant/core/issues/37758

Setting the HVAC_MODE turns it on, sentind turn_on service does nothing, even on the lovelace it is irresponsive.

Reply:
Hm, have a look in the pydaikin code I linked to above, that will turn on if mode isn't `off`. The fix should be implemented there and this solution with setting motde to `auto` is not desirable.  (the unit and pydaikin knows the last state and we like to use that) .